website/source/docs/guides/consul-aws.html.md

```diff
@@ -0,0 +1,88 @@
+This guide will describe how to configure and how to start the sync.
+
+## Authentication
+
+`consul-aws` need access to Consul and AWS for uni- and bidirectional sync.
+
+For Consul, the process accepts both the standard CLI flags, `-token` and the environment variables `CONSUL_HTTP_TOKEN`. This should be set to an Consul ACL token if ACLs are enabled.
+
+For AWS, `consul-aws` uses the default credential provider chain to find AWS credentials. The default provider chain looks for credentials in the following order:
+1. Environment variables.
+2. Shared credentials file.
+3. If your application is running on an Amazon EC2 instance, IAM role for Amazon EC2.
+
+## Configuration
+
+There are two subcommands available on `consul-aws`:
+
+* version: display version number
+* sync-catalog: start syncing the catalogs
+
+The version subcommand doesn’t do anything besides showing the version, so lets focus on sync-catalog. The following flags are available:
+
+* A set of parameters to connect to your Consul Cluster like `-http-addr`, `-token`, `-ca-file`, `-client-cert`, and everything else you might need in order to do that
+* `-aws-namespace-id`: The AWS namespace to sync with Consul services.
+* `-aws-service-prefix`: A prefix to prepend to all services written to AWS from Consul. If this is not set then services will have no prefix.
+* `-consul-service-prefix`: A prefix to prepend to all services written to Consul from AWS. If this is not set then services will have no prefix.
+* `-to-aws`: If true, Consul services will be synced to AWS (defaults to false).
+* `-to-consul`: If true, AWS services will be synced to Consul (defaults to false).
+* `-aws-pull-interval`: The interval between fetching from AWS CloudMap. Accepts a sequence of decimal numbers, each with optional fraction and a unit suffix, such as "300ms", "10s", "1.5m" (defaults to 30s).
+* `-aws-dns-ttl`: DNS TTL for services created in AWS CloudMap in seconds (defaults to 60).
+
+Independent of how you want to user `consul-aws` it needs to be able to connect to Consul and AWS. Apart from making sure you setup up authenticated access, `-aws-namespace-id` is mandatory.
+
+## Syncing Consul services to AWS CloudMap
+
+Assuming authenticated access is set up, there is little left to do before starting the sync. Using `-to-aws` command line flag will start the sync to AWS CloudMap. If `-aws-service-prefix` is provided, every imported service from Consul will be prefixed. For example:
+
+```shell
+$ consul-aws -aws-namespace-id ns-hjrgt3bapp7phzff -to-aws -consul-service-prefix consul_
+```
+
+At this point `consul-aws` will start importing services into AWS Route. A service in Consul named `web` will end up becoming `consul_web` in AWS. The individual services instances from Consul will be created in AWS as well.
```

Comment:
Should this be Route53?

> services into AWS Route

Or should it use AWS CloudMap?

The individual services instances > The individual **service** instances (not sure?)